<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="dhanushka-ThinkPad-T530" name="org.wso2.extension.siddhi.io.snmp.sink.TestCaseOfSNMPSinkV3" tests="5" failures="0" timestamp="24 Feb 2019 03:43:40 GMT" time="2.366" errors="0">
  <testcase name="snmpVersion3BasicSink" time="1.267" classname="org.wso2.extension.siddhi.io.snmp.sink.TestCaseOfSNMPSinkV3"/>
  <testcase name="snmpVersion3SinkException" time="0.008" classname="org.wso2.extension.siddhi.io.snmp.sink.TestCaseOfSNMPSinkV3"/>
  <testcase name="snmpVersion3SinkCreationException" time="0.016" classname="org.wso2.extension.siddhi.io.snmp.sink.TestCaseOfSNMPSinkV3"/>
  <testcase name="snmpVersion3TCPSink" time="1.075" classname="org.wso2.extension.siddhi.io.snmp.sink.TestCaseOfSNMPSinkV3"/>
  <testcase name="snmpVersion3SinkValidationException" classname="org.wso2.extension.siddhi.io.snmp.sink.TestCaseOfSNMPSinkV3">
    <skipped/>
  </testcase> <!-- snmpVersion3SinkValidationException -->
</testsuite> <!-- org.wso2.extension.siddhi.io.snmp.sink.TestCaseOfSNMPSinkV3 -->
